buildSrc/mac.gradle

```diff
@@ -55,8 +55,7 @@ def defaultSdkPath = "/Applications/Xcode.app/Contents/Developer/Platforms/MacOS
 
 // Set the minimum API version that we require (developers do not need to override this)
 // Note that this is not necessarily the same as the preferred SDK version
-def isAarch64 = TARGET_ARCH == "aarch64" || TARGET_ARCH == "arm64";
-def macOSMinVersion = isAarch64 ? "11.0" : "10.12";
+def macOSMinVersion = "11.0";
```

Comment:
is the intent here to always be in sync?
or do you see a possible situation where we would need to have target-specific conditionals again?

Reply:
> is the intent here to always be in sync?

Yes. Having a different minimum was a one-time anomaly due to the fact that Apple required a minimum target of 11.0 for M1 systems when they introduce them. See [JDK-8265031](https://bugs.openjdk.org/browse/JDK-8265031).

> do you see a possible situation where we would need to have target-specific conditionals again?

No.